oeps/processes/oep-0055/decisions/0003-readme-specification.rst

```diff
@@ -0,0 +1,209 @@
+0003 README specification
+#########################
+
+Status
+******
+
+** Accepted **
+
+Context
```

Comment:
Oops. @feanil - I didn't mean to approve without https://github.com/openedx/open-edx-proposals/pull/363#discussion_r936684013 being addressed. It was mistakenly marked as outdated, because a separate change came through on the same line.

Reply:
Ah, yes, I meant to reply on that one, I just added an update to OEP-55 and OEP-19

Reply:
Thanks. I think the context paragraph should note the history of addressing READMEs in OEP-19, but I won't belabor the point. I understand it wasn't part of your immediate concern, OEP-55, but it was the initial attempt at codifying the use and purpose of a README.